helm/alfresco-content-services/templates/config-ooi-service.yaml

```diff
@@ -0,0 +1,17 @@
+{{- if .Values.ooi.enabled }}
```

Comment:
Is OOI available for Community too?

If not, it might also be worth adding a check for Enterprise edition:

`{{- if eq .Values.repository.edition "Enterprise" }}`

as used in helm/alfresco-content-services/templates/deployment-transform-router.yaml

Reply:
No, not Community, sorry. Note: ADW and OOI are currently Enterprise-only.

Reply:
Added the extra check.